File: divisor-java-springboot-mvc/src/main/java/my/test/divisor/springboot/mvc/config/ApplicationConfiguration.java
package my.test.divisor.springboot.mvc.config;

import my.test.divisor.springboot.mvc.domain.service.DivisorService;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class ApplicationConfiguration {

    private static final int DIVISOR_RANGE_LOWER_END = 1;
    private static final int DIVISOR_RANGE_UPPER_END = 10;

    @Bean
    public DivisorService divisorService() {
        return new DivisorService(DIVISOR_RANGE_LOWER_END, DIVISOR_RANGE_UPPER_END);
    }
}

File: divisor-kotlin-springboot-mvc/src/main/kotlin/my/test/divisor/kotlin/springboot/mvc/config/ApplicationConfiguration.kt
package my.test.divisor.kotlin.springboot.mvc.config

import my.test.divisor.kotlin.springboot.mvc.domain.service.DivisorService
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

private val DIVISOR_RANGE_LOWER_END = 1
private val DIVISOR_RANGE_UPPER_END = 10

@Configuration
class ApplicationConfiguration {

    @Bean
    fun divisorService() = DivisorService(DIVISOR_RANGE_LOWER_END, DIVISOR_RANGE_UPPER_END)
}